kafka-ui-react-app/src/components/Topics/Topic/Details/Overview/__test__/Overview.spec.tsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
+import { render } from 'lib/testHelpers';
 import Overview from 'components/Topics/Topic/Details/Overview/Overview';
+import { Colors } from 'theme/theme';
 
 describe('Overview', () => {
   const mockInternal = false;
```

Comment:
Maybe we should move this constants to fixtures. And also create an object rather then many strings, type this object with some interface(or use one from Overview component props) and pass as props into renderComponent